llvm/docs/LangRef.rst

```diff
@@ -14389,6 +14389,34 @@ Semantics:
 
 Note this intrinsic is only verified on AArch64 and ARM.
 
+'``llvm.stackaddress``' Intrinsic
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Syntax:
+"""""""
+
+::
+
+      declare ptr @llvm.stackaddress()
+
+Overview:
+"""""""""
+
+The '``llvm.stackaddress``' instrinsic returns the starting address of the stack region that may be
+used by called functions.
```

Comment:
The text here looks over-wide. LLVM usually uses 80 columns, including for docs.